diff --git a/src/Remote/Button/ChannelDownButton.php b/src/Remote/Button/ChannelDownButton.php
index 1417bda..054062f 100644
--- a/src/Remote/Button/ChannelDownButton.php
+++ b/src/Remote/Button/ChannelDownButton.php
@@ -2,9 +2,12 @@
 
 namespace App\Remote\Button;
 
+use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
+#[AsTaggedItem('channel-down')]
 final class ChannelDownButton implements ButtonInterface
 {
     public function press(): void
diff --git a/src/Remote/Button/ChannelUpButton.php b/src/Remote/Button/ChannelUpButton.php
index 44ffa89..7936059 100644
--- a/src/Remote/Button/ChannelUpButton.php
+++ b/src/Remote/Button/ChannelUpButton.php
@@ -2,9 +2,12 @@
 
 namespace App\Remote\Button;
 
+use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
+#[AsTaggedItem('channel-up')]
 final class ChannelUpButton implements ButtonInterface
 {
     public function press(): void
diff --git a/src/Remote/Button/PowerButton.php b/src/Remote/Button/PowerButton.php
index 92b67d9..b45b6b4 100644
--- a/src/Remote/Button/PowerButton.php
+++ b/src/Remote/Button/PowerButton.php
@@ -2,9 +2,12 @@
 
 namespace App\Remote\Button;
 
+use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
+#[AsTaggedItem('power')]
 final class PowerButton implements ButtonInterface
 {
     public function press(): void
diff --git a/src/Remote/Button/VolumeDownButton.php b/src/Remote/Button/VolumeDownButton.php
index c82e9e1..0066fc1 100644
--- a/src/Remote/Button/VolumeDownButton.php
+++ b/src/Remote/Button/VolumeDownButton.php
@@ -2,9 +2,12 @@
 
 namespace App\Remote\Button;
 
+use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
+#[AsTaggedItem('volume-down')]
 final class VolumeDownButton implements ButtonInterface
 {
     public function press(): void
diff --git a/src/Remote/Button/VolumeUpButton.php b/src/Remote/Button/VolumeUpButton.php
index 58ccffe..16bb26c 100644
--- a/src/Remote/Button/VolumeUpButton.php
+++ b/src/Remote/Button/VolumeUpButton.php
@@ -2,9 +2,12 @@
 
 namespace App\Remote\Button;
 
+use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
+#[AsTaggedItem('volume-up')]
 final class VolumeUpButton implements ButtonInterface
 {
     public function press(): void
