diff --git a/src/Remote/Button/ButtonInterface.php b/src/Remote/Button/ButtonInterface.php
index aabab04..b5808da 100644
--- a/src/Remote/Button/ButtonInterface.php
+++ b/src/Remote/Button/ButtonInterface.php
@@ -2,9 +2,12 @@
 
 namespace App\Remote\Button;
 
+use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
+#[AutoconfigureTag]
 interface ButtonInterface
 {
     public function press(): void;
