diff --git a/src/Controller/RemoteController.php b/src/Controller/RemoteController.php
index ff5cdb0..9e6a0de 100644
--- a/src/Controller/RemoteController.php
+++ b/src/Controller/RemoteController.php
@@ -2,6 +2,8 @@
 
 namespace App\Controller;
 
+use App\Remote\ButtonRemote;
+use Psr\Container\NotFoundExceptionInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
@@ -14,27 +16,13 @@ use function Symfony\Component\String\u;
 final class RemoteController extends AbstractController
 {
     #[Route('/', name: 'home', methods: ['GET', 'POST'])]
-    public function index(Request $request): Response
+    public function index(Request $request, ButtonRemote $remote): Response
     {
         if ('POST' === $request->getMethod()) {
-            switch ($button = $request->request->getString('button')) {
-                case 'power':
-                    dump('Power on/off the TV');
-                    break;
-                case 'channel-up':
-                    dump('Change the channel up');
-                    break;
-                case 'channel-down':
-                    dump('Change the channel down');
-                    break;
-                case 'volume-up':
-                    dump('Increase the volume');
-                    break;
-                case 'volume-down':
-                    dump('Decrease the volume');
-                    break;
-                default:
-                    throw $this->createNotFoundException(sprintf('Button "%s" not found.', $button));
+            try {
+                $remote->press($button = $request->request->getString('button'));
+            } catch (NotFoundExceptionInterface) {
+                throw $this->createNotFoundException(sprintf('Button "%s" not found.', $button));
             }
 
             $this->addFlash('success', sprintf('%s pressed', u($button)->replace('-', ' ')->title(allWords: true)));
