diff --git a/src/Remote/ButtonRemote.php b/src/Remote/ButtonRemote.php
index e522005..b5c9bc1 100644
--- a/src/Remote/ButtonRemote.php
+++ b/src/Remote/ButtonRemote.php
@@ -11,7 +11,7 @@ use Symfony\Component\DependencyInjection\Attribute\AutowireIterator;
 final class ButtonRemote
 {
     public function __construct(
-        #[AutowireIterator(ButtonInterface::class)]
+        #[AutowireIterator(ButtonInterface::class, indexAttribute: 'key')]
         private iterable $buttons,
     ) {
     }
