diff --git a/src/Controller/RemoteController.php b/src/Controller/RemoteController.php
index 56b8613..a623438 100644
--- a/src/Controller/RemoteController.php
+++ b/src/Controller/RemoteController.php
@@ -27,8 +27,6 @@ final class RemoteController extends AbstractController
             return $this->redirectToRoute('home');
         }
 
-        dd($remote->buttons());
-
         return $this->render('index.html.twig', [
             'buttons' => $remote->buttons(),
         ]);
