diff --git a/src/Remote/Button/DiagnosticsButton.php b/src/Remote/Button/DiagnosticsButton.php
index f601d3f..5124a8c 100644
--- a/src/Remote/Button/DiagnosticsButton.php
+++ b/src/Remote/Button/DiagnosticsButton.php
@@ -3,10 +3,12 @@
 namespace App\Remote\Button;
 
 use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+use Symfony\Component\DependencyInjection\Attribute\Exclude;
 use Symfony\Component\DependencyInjection\Attribute\When;
 
 #[AsTaggedItem('diagnostics')]
 #[When('dev')]
+#[Exclude]
 final class DiagnosticsButton implements ButtonInterface
 {
     public function press(): void
