diff --git a/src/Controller/RemoteController.php b/src/Controller/RemoteController.php
index bb418f8..5bf3f7f 100644
--- a/src/Controller/RemoteController.php
+++ b/src/Controller/RemoteController.php
@@ -7,6 +7,7 @@ use App\Remote\Button\PowerButton;
 use App\Remote\RemoteInterface;
 use Psr\Container\NotFoundExceptionInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
@@ -38,8 +39,10 @@ final class RemoteController extends AbstractController
     }
 
     #[Route('/power', name: 'power')]
-    public function power(ButtonInterface $button): Response
-    {
+    public function power(
+        #[Autowire(service: PowerButton::class)]
+        ButtonInterface $button,
+    ): Response {
         if (true) { // some condition
             dd($button);
         }
