diff --git a/src/Remote/Button/PowerButton.php b/src/Remote/Button/PowerButton.php
index 74d4a86..3910cd2 100644
--- a/src/Remote/Button/PowerButton.php
+++ b/src/Remote/Button/PowerButton.php
@@ -7,7 +7,7 @@ use Symfony\Component\DependencyInjection\Attribute\Lazy;
 
 #[AsTaggedItem('power', priority: 50)]
 #[Lazy]
-final class PowerButton implements ButtonInterface
+class PowerButton implements ButtonInterface
 {
     public function press(): void
     {
