diff --git a/src/Remote/Button/PowerButton.php b/src/Remote/Button/PowerButton.php
index b74b86b..56071ac 100644
--- a/src/Remote/Button/PowerButton.php
+++ b/src/Remote/Button/PowerButton.php
@@ -3,8 +3,10 @@
 namespace App\Remote\Button;
 
 use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
+use Symfony\Component\DependencyInjection\Attribute\Lazy;
 
 #[AsTaggedItem('power', priority: 100)]
+#[Lazy]
 final class PowerButton implements ButtonInterface
 {
     public function press(): void
