diff --git a/src/Remote/ParentalControls.php b/src/Remote/ParentalControls.php
index 6360e59..94a7c4d 100644
--- a/src/Remote/ParentalControls.php
+++ b/src/Remote/ParentalControls.php
@@ -2,8 +2,10 @@
 
 namespace App\Remote;
 
+use Symfony\Component\DependencyInjection\Attribute\Lazy;
 use Symfony\Component\Mailer\MailerInterface;
 
+#[Lazy]
 final class ParentalControls
 {
     public function __construct(
