diff --git a/src/Remote/Button/VolumeUpButton.php b/src/Remote/Button/VolumeUpButton.php
index 74d1227..f9e503b 100644
--- a/src/Remote/Button/VolumeUpButton.php
+++ b/src/Remote/Button/VolumeUpButton.php
@@ -2,13 +2,23 @@
 
 namespace App\Remote\Button;
 
+use App\Remote\ParentalControls;
 use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
 
 #[AsTaggedItem('volume-up', priority: 20)]
 final class VolumeUpButton implements ButtonInterface
 {
+    public function __construct(
+        private ParentalControls $parentalControls,
+    ) {
+    }
+
     public function press(): void
     {
+        if (true) { // detect if volume is too high
+            $this->parentalControls->volumeToHigh();
+        }
+
         dump('Change the volume up');
     }
 }
