diff --git a/src/Remote/Button/ButtonInterface.php b/src/Remote/Button/ButtonInterface.php
index af97822..4f5b318 100644
--- a/src/Remote/Button/ButtonInterface.php
+++ b/src/Remote/Button/ButtonInterface.php
@@ -3,8 +3,10 @@
 namespace App\Remote\Button;
 
 use Symfony\Component\DependencyInjection\Attribute\AutoconfigureTag;
+use Symfony\Component\DependencyInjection\Attribute\Lazy;
 
 #[AutoconfigureTag]
+#[Lazy(ButtonInterface::class)]
 interface ButtonInterface
 {
     public function press(): void;
diff --git a/src/Remote/Button/PowerButton.php b/src/Remote/Button/PowerButton.php
index 0760549..8bc1e05 100644
--- a/src/Remote/Button/PowerButton.php
+++ b/src/Remote/Button/PowerButton.php
@@ -3,10 +3,8 @@
 namespace App\Remote\Button;
 
 use Symfony\Component\DependencyInjection\Attribute\AsTaggedItem;
-use Symfony\Component\DependencyInjection\Attribute\Lazy;
 
 #[AsTaggedItem('power', priority: 50)]
-#[Lazy(ButtonInterface::class)]
 final class PowerButton implements ButtonInterface
 {
     public function press(): void
