diff --git a/src/Remote/LoggerRemote.php b/src/Remote/LoggerRemote.php
index 3bd30ef..18ce5d0 100644
--- a/src/Remote/LoggerRemote.php
+++ b/src/Remote/LoggerRemote.php
@@ -3,7 +3,9 @@
 namespace App\Remote;
 
 use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\Attribute\AsDecorator;
 
+#[AsDecorator(RemoteInterface::class)]
 final class LoggerRemote implements RemoteInterface
 {
     public function __construct(
