diff --git a/src/Remote/ButtonRemote.php b/src/Remote/ButtonRemote.php
index a97dfd6..8ea3598 100644
--- a/src/Remote/ButtonRemote.php
+++ b/src/Remote/ButtonRemote.php
@@ -9,7 +9,7 @@ use Symfony\Contracts\Service\ServiceCollectionInterface;
 /**
  * @author Kevin Bond <kevinbond@gmail.com>
  */
-final class ButtonRemote
+final class ButtonRemote implements RemoteInterface
 {
     public function __construct(
         #[AutowireLocator(ButtonInterface::class, indexAttribute: 'key')]
@@ -22,9 +22,6 @@ final class ButtonRemote
         $this->buttons->get($name)->press();
     }
 
-    /**
-     * @return string[]
-     */
     public function buttons(): iterable
     {
         return array_keys($this->buttons->getProvidedServices());
diff --git a/src/Remote/RemoteInterface.php b/src/Remote/RemoteInterface.php
new file mode 100644
index 0000000..86ba2ee
--- /dev/null
+++ b/src/Remote/RemoteInterface.php
@@ -0,0 +1,16 @@
+<?php
+
+namespace App\Remote;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+interface RemoteInterface
+{
+    public function press(string $name): void;
+
+    /**
+     * @return string[]
+     */
+    public function buttons(): iterable;
+}
