diff --git a/src/Controller/RemoteController.php b/src/Controller/RemoteController.php
index d88a9b7..fb0f9fd 100644
--- a/src/Controller/RemoteController.php
+++ b/src/Controller/RemoteController.php
@@ -8,6 +8,7 @@ use App\Remote\RemoteInterface;
 use Psr\Container\NotFoundExceptionInterface;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\DependencyInjection\Attribute\Autowire;
+use Symfony\Component\DependencyInjection\Attribute\AutowireServiceClosure;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Attribute\Route;
@@ -50,15 +51,19 @@ final class RemoteController extends AbstractController
         }
     }
 
+    /**
+     * @param \Closure():ButtonInterface $remote
+     */
     #[Route('/buttons', name: 'buttons')]
     public function buttons(
-        RemoteInterface $remote,
+        #[AutowireServiceClosure(RemoteInterface::class)]
+        \Closure $remote,
         CacheInterface $cache,
     ): Response {
         $buttons = $cache->get('buttons', function () use ($remote) {
-            dd($remote->buttons());
+            dd($remote()->buttons());
 
-            return $remote->buttons();
+            return $remote()->buttons();
         });
 
         return $this->json($buttons);
