diff --git a/src/Remote/ButtonRemote.php b/src/Remote/ButtonRemote.php
index 4ce8faa..a97dfd6 100644
--- a/src/Remote/ButtonRemote.php
+++ b/src/Remote/ButtonRemote.php
@@ -27,8 +27,6 @@ final class ButtonRemote
      */
     public function buttons(): iterable
     {
-        foreach ($this->buttons as $name => $button) {
-            yield $name;
-        }
+        return array_keys($this->buttons->getProvidedServices());
     }
 }
