diff --git a/src/Remote/LoggerRemote.php b/src/Remote/LoggerRemote.php
index 3b37bd0..b9449a9 100644
--- a/src/Remote/LoggerRemote.php
+++ b/src/Remote/LoggerRemote.php
@@ -11,7 +11,7 @@ use Symfony\Component\DependencyInjection\Attribute\Target;
 final class LoggerRemote implements RemoteInterface
 {
     public function __construct(
-        #[Target('buttonLogger')]
+        #[Target('buttonsLogger')]
         private LoggerInterface $logger,
 
         #[AutowireDecorated]
