diff --git a/src/Remote/LoggerRemote.php b/src/Remote/LoggerRemote.php
index 18ce5d0..4fceaf6 100644
--- a/src/Remote/LoggerRemote.php
+++ b/src/Remote/LoggerRemote.php
@@ -4,12 +4,15 @@ namespace App\Remote;
 
 use Psr\Log\LoggerInterface;
 use Symfony\Component\DependencyInjection\Attribute\AsDecorator;
+use Symfony\Component\DependencyInjection\Attribute\Target;
 
 #[AsDecorator(RemoteInterface::class)]
 final class LoggerRemote implements RemoteInterface
 {
     public function __construct(
         private RemoteInterface $inner,
+
+        #[Target('buttonLogger')]
         private LoggerInterface $logger,
     ) {
     }
