diff --git a/src/Remote/LoggerRemote.php b/src/Remote/LoggerRemote.php
index 24bbc76..3b37bd0 100644
--- a/src/Remote/LoggerRemote.php
+++ b/src/Remote/LoggerRemote.php
@@ -5,12 +5,14 @@ namespace App\Remote;
 use Psr\Log\LoggerInterface;
 use Symfony\Component\DependencyInjection\Attribute\AsDecorator;
 use Symfony\Component\DependencyInjection\Attribute\AutowireDecorated;
+use Symfony\Component\DependencyInjection\Attribute\Target;
 
 #[AsDecorator(ButtonRemote::class)]
 final class LoggerRemote implements RemoteInterface
 {
     public function __construct(
-        private LoggerInterface $buttonLogger,
+        #[Target('buttonLogger')]
+        private LoggerInterface $logger,
 
         #[AutowireDecorated]
         private RemoteInterface $inner,
@@ -19,11 +21,11 @@ final class LoggerRemote implements RemoteInterface
 
     public function press(string $name): void
     {
-        $this->buttonLogger->info('Pressing button '.$name);
+        $this->logger->info('Pressing button '.$name);
 
         $this->inner->press($name);
 
-        $this->buttonLogger->info('Pressed button '.$name);
+        $this->logger->info('Pressed button '.$name);
     }
 
     /**
