diff --git a/src/Remote/LoggerRemote.php b/src/Remote/LoggerRemote.php
index 85255d9..75e2cf8 100644
--- a/src/Remote/LoggerRemote.php
+++ b/src/Remote/LoggerRemote.php
@@ -13,17 +13,17 @@ final class LoggerRemote implements RemoteInterface
 {
     public function __construct(
         private RemoteInterface $inner,
-        private LoggerInterface $buttonLogger,
+        private LoggerInterface $logger,
     ) {
     }
 
     public function press(string $name): void
     {
-        $this->buttonLogger->info('Pressing button {name}', ['name' => $name]);
+        $this->logger->info('Pressing button {name}', ['name' => $name]);
 
         $this->inner->press($name);
 
-        $this->buttonLogger->info('Button {name} pressed', ['name' => $name]);
+        $this->logger->info('Button {name} pressed', ['name' => $name]);
     }
 
     public function buttons(): iterable
